docs/src/pages/careers/lead-designer.md

```diff
@@ -65,26 +68,42 @@ As a team leader, you will:
-- **Enough development skills to implement the designed interfaces**. Our React engineers are often busy implementing complex components. We are looking for somebody with enough CSS-in-JS and JSX skills to be able to deeply understand what's the process like. The objective is to a. better understand the product, b. improve the collaboration with the engineers (the implementation is all that matters for end-users) and c. occasionally not depend on them to implement the design.
-- **Strong written and verbal communication skills**. As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
-- **Management of a design team**. Whether formally or informally, you have shown that you are a design leader and can provide guidance and support to other designers.
+- **Expert-level UI knowledge**. You can craft great-looking designs, spotting tiny details most people wouldn't pay attention to.
+- **Expert-level UX knowledge**. You can build intuitive solutions, both high and low-level ones. You care about the entire user journey as well as an interaction with a specific component.
+- **Sharp written and verbal communication skills**. Communicating clearly is fundamental to create and manage high-quality products and teams. You'll interface with our team, community members, enterprise customers and also contribute to user documentation.
+- **Managerial abilities**. You have either managed a team before or are feeling ready to start now. You've shown that you're a design leader at heart and can provide the needed guidance and support to other designers.
+- **Comfortable around design systems**. You've worked on one before and have experienced the challenges of developing it.
+- **Enough development autonomy**. Our products are essentially code. Knowing how to navigate through code is key to understanding MUI products and collaborating with engineers. You know how to work with CSS (mainly CSS-in-JS), JSX, and Git.
 
 ### What would be nice if you had, but isn't required
 
-- **You've contributed to a design system before**. You have worked on a library of components in the past, maybe for internal usage in your company.
+- **Experience with open-source products**. You have seen the particular product dynamics when in an OSS environment and are knowledgable of the challenges.
 
 ## Benefits & Compensation
 
 Competitive compensation depending on the profile and location.
 We are ready to pay top market rates for a person that can clearly exceed the role's expectations.
 You can find the other perks & benefits on the [careers](https://mui.com/careers/#perks-amp-benefits) page.
 
+## Interview steps
+
+Here's a brief overview of how this role's process will be carried through:
+
+- **Introduction call** (20 to 25 min ⎯ with a designer): A short call to quickly introduce each other.
```

Comment:
How about?

```suggestion
- **Introduction call** (20 to 25 min): A short call to quickly introduce each other.
```

It took 53 and 72 for the last two hires [source](https://app.ashbyhq.com/reports/saved/5d950db9-9c0f-4099-be11-1ddbc41c3e1e) but ashby reports 4 interviews done with designers in the last 30 days: https://app.ashbyhq.com/reports/saved/b56119d5-ac73-4117-9398-2004685cff15. So at this pace, we would need   375 days to fill the role 🙃.